memory/memory/memory.py

```diff
@@ -70,15 +72,14 @@ async def get_or_add_user(app, userdata):
     username = userdata['username']
     if username not in users:
         k8s_client = app['k8s_client']
-        hail_identity_secret = await retry_transient_errors(
+        hail_credentials_secret_name = await retry_transient_errors(
```

Comment:
Can you remove the `_name` from the suffix here? You're returning a secret and not a name.